ui/shared/utils/httpRequestHelper.js

```diff
@@ -36,12 +36,12 @@ export class HttpRequestHelper {
    * @param urlParams A dictionary of key-value pairs {gene: 'ENSG00012345', chrom: '1'} to encode
    *   and append to the url as HTTP GET params (eg. "?gene=ENSG00012345&chrom=1")
    */
-  get = (urlParams = {}) => {
+  get = (urlParams = {}, credentials = 'include') => {
     const urlQueryString = getUrlQueryString(urlParams)
 
     const p = fetch(`${this.url}?${urlQueryString}`, {
       method: 'GET',
-      credentials: 'include',
+      credentials,
```

Comment:
I think it would be clearer/more extensible if instead of having an optional `credentials` param we instead allow an optional dictionary of `fetchParams` that we add here - 
```
method: 'GET',
credentials: 'include',
...(fetchParams || {})
```

 This way the VCI request would have `{ credentials: 'omit' }` as the param it passes which is more readable

Reply:
Sounds good.